src/relay.js

```diff
@@ -233,6 +233,23 @@ export function sequelizeConnection({name, nodeType, target, orderBy: orderByEnu
           model.sequelize.literal('COUNT(1) OVER()'),
           'full_count'
         ]);
+      } else if (options.limit) {
+        // TODO: scope are not applied to the where clause!
+
+        const countQuery = model
```

Comment:
Nice catch!
The code will look like:
```js
const countOptions = {
  attributes: [['COUNT(*)', 'full_count']],
  include: options.include,
  where: options.where
}

model.$conformOptions(options, model);
model.$injectScope(options);
model.$expandIncludeAll(options);
model.$validateIncludedElements(options);
model._paranoidClause(options);

const countQuery = model
  .QueryInterface
  .QueryGenerator
  .selectQuery(model.tableName, countOptions, model)
  .replace(/;$/, '');
```

I support that methods prefixed with `$` are not "private" but `_paranoidClause` is... Base our implementation on top of private methods is not a good idea...

I tried an other approach: fetch count with `model.count()` but I didn't find a way to pass the result to `after`.